### PR TITLE
some suggestions for moodle 3.2

### PR DESCRIPTION
**Pull request by mwuttke**
Hello Maxim,

here are some suggestions for the new moodle 3.2 version, which will be released probably tomorrow: the travis-ci intergration (removed PHP 5.4/5.5 & added master/Moodle 3.2 checks) and changes in the version.php, the README.md, the CHANGELOG.md and in the build.xml.

Thanks & Greetings,
Michael

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,15 +3,15 @@
 This Moodle plugin uses Apache OpenMeetings.
 You need your own Apache OpenMeetings instance running.
 
-[![Build Status](https://travis-ci.org/openmeetings/openmeetings-moodle-plugin.svg?branch=master)](https://travis-ci.org/openmeetings/openmeetings-moodle-plugin)
+[![Build Status](https://travis-ci.org/moodlebeuth/moodle-mod_openmeetings.svg?branch=master)](https://travis-ci.org/moodlebeuth/moodle-mod_openmeetings)
```
Comment (solomax) [inline README.md:6]: Why this change is required?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 Apache OpenMeetings Moodle Plugin Change Log
 
+2.0.0.6 (2016112701)
+ * Moodle plugin is updated to work as expected with OM 3.2
```
Comment (solomax) [inline CHANGELOG.md:4]: I guess it need to be "as expected with Moodle 3.2"

File: lang/ru/openmeetings.php [hunk 3]
```diff
@@ -35,65 +35,66 @@
...
 
-$string['description'] = '&#1054;&#1087;&#1080;&#1089;&#1072;&#1085;&#1080;&#1077;';
+$string['description'] = 'Описание';
 
-$string['Chat_Hidden'] = '&#0421;&#043F;&#0440;&#044F;&#0442;&#0430;&#0442;&#044C; &#0427;&#0430;&#0442;';
-$string['Chat_Hidden_TYPE_1'] = '&#041F;&#043E;&#043A;&#0430;&#0437;&#0430;&#0442;&#044C; &#0447;&#0430;&#0442; &#0432; &#043A;&#043E;&#043C;&#043D;&#0430;&#0442;&#0435;';
-$string['Chat_Hidden_TYPE_2'] = '&#0421;&#043F;&#0440;&#044F;&#0442;&#0430;&#0442;&#044C; &#0447;&#0430;&#0442; &#0432; &#043A;&#043E;&#043C;&#043D;&#0430;&#0442;&#0435;';
+$string['Chat_Hidden'] = 'Отображение чата';
+$string['Chat_Hidden_TYPE_1'] = 'Показывать чат комнаты';
+$string['Chat_Hidden_TYPE_2'] = 'Скрывать чат комнаты';
```
Comment (solomax) [inline lang/ru/openmeetings.php:100]: I was sure I have already merged these changes .....

**Comment by mwuttke** [comment]
Yes, you are right. Can you change the README and the CHANGELOG again and ignore your previous changes in russian language file?
Michael

File: version.php [hunk 4]
```diff
@@ -36,10 +36,10 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-$plugin->version    = 2016052401;
+$plugin->version    = 2016112701;
 $plugin->component  = 'mod_openmeetings';
 $plugin->maturity   = MATURITY_STABLE;
-$plugin->requires   = 2014051200;
-$plugin->release    = '2.0.0.4';
+$plugin->requires   = 2016112500;
```
Comment (solomax) [inline version.php:42]: I see no changes why this plugin is no more compatible with older Moodle ....
Could you please explain it?

**Comment by solomax** [comment]
Partially merged

**Comment by solomax** [comment]
https://github.com/openmeetings/openmeetings-moodle-plugin/commit/b413142158e23083283bd92d33d16ab4a884aef4

File: version.php (hunk 4 above)
Comment (mwuttke) [reply version.php:42]: Sorry for mess things up. The upcoming Moodle 3.2 needs php 7.0.

Thanks,
Michael

Comment (solomax) [reply version.php:42]: OK, But since PHP code was not changed, plugin would work with 
Moodle 3.0, PHP 5
as well as with 
Moodle 3.2, PHP 7

I see no need to change `requires` field and drop 3.0 support ....

Comment (mwuttke) [reply version.php:42]: Ok

**Comment by solomax** [comment]
Closing PR :)
Thanks for the contribution :)
